elytron/src/test/resources/org/wildfly/extension/elytron/tls-test.xml

```diff
@@ -23,21 +23,25 @@
     </credential-stores>
     <tls>
         <key-stores>
-            <key-store name="FireflyKeystore" type="JKS">
-                <file path="firefly.keystore" relative-to="jboss.server.config.dir"/>
+            <key-store name="FireflyKeystore" >
                 <credential-reference store="test" alias="master-password-alias"/>
+                <implementation type="JKS" />
```

Comment:
To follow the model configuration.
you have attribute group "implementation" that has this attributes on it.
It is present in model and as such grouped in admin console, so it makes sense to have it grouped also in xml.

on same resource there are other 3 attributes that are in group "file" which is already used also in xml. So for consistency sake all attribute groups are now persisted the same.
We could change parser config to not use elements for groups but in that case file attributes would follow.